src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -59,7 +59,11 @@ public function normalize($object, $format = null, array $context = array())
         $timezone = $this->getTimezone($context);
 
         if (null !== $timezone) {
-            $object = (new \DateTimeImmutable('@'.$object->getTimestamp()))->setTimezone($timezone);
+            if ($object instanceof \DateTime) {
+                $object = clone $object;
+            }
+
+            $object = $object->setTimezone($timezone);
```

Comment:
mutating the original datetime is a change, but we safely clone any datetime no? https://3v4l.org/gdLh0

Reply:
in clear, you the change should be just:
`$object = (clone $object)->setTimezone($timezone);`

Reply:
@nicolas-grekas i can't do it in one line because php 5.5 don't support it